apps/sensenet/src/components/view-controls/permission-view.tsx

```diff
@@ -42,9 +42,11 @@ const useStyles = makeStyles((theme: Theme) => {
     },
     title: {
       fontSize: '20px',
+      paddingRight: '10px',
     },
-    textBolder: {
+    contentName: {
       fontWeight: 500,
+      wordBreak: 'break-word',
```

Comment:
it is working with the default value (normal) of wordBreak, so you can delete this

Reply:
but it is not working with some display size 
I think it is better with break
![permission_button](https://user-images.githubusercontent.com/22347393/95055298-5fdbec00-06f3-11eb-93e1-78da0314c333.gif)